qiskit_experiments/library/characterization/fine_amplitude.py

```diff
@@ -127,7 +127,7 @@ def __init__(
         """
         super().__init__(qubits, analysis=FineAmplitudeAnalysis(), backend=backend)
         self.set_experiment_options(gate=gate)
-        self._measurement_qubits = measurement_qubits or qubits
+        self._measurement_qubits = measurement_qubits or range(len(qubits))
```

Comment:
Since superclass is already initialized, can we use `self.num_qubits` instead? Probably I would change this logic. See the comment above.

Reply:
See here: [190e0ed](https://github.com/Qiskit/qiskit-experiments/pull/698/commits/190e0edfbc0fadcf9794d5a07adac8fcd50b0d5d)